controller/src/beerocks/bml/bml.cpp

```diff
@@ -686,7 +686,7 @@ int bml_get_dcs_scan_results(BML_CTX ctx, const char *radio_mac,
                              unsigned int *output_results_size, unsigned char *output_result_status,
                              bool is_single_scan)
 {
-    // Validate intput params;
+    // Validate input parameters
```

Comment:
in the commit message:
just write down `PPM-5`
or
`Relates to JIRA PPM-5`

but for sure don't mention the PR.

Reply:
Updated